holodeck/environments.py

```diff
@@ -275,6 +275,20 @@ def spawn_agent(self, agent_definition, location):
         self._add_agents(agent_definition)
         self._enqueue_command(SpawnAgentCommand(location, agent_definition.name, agent_definition.type))
 
+    def set_ticks_per_capture(self, agent_name, ticks_per_capture):
+        """Queues a rgb camera rate command. It will be applied when `tick` or `step` is called next.
+        The specified agent's rgb camera will capture images every specified number of ticks.
+        The sensor's image will remain unchanged between captures.
+        This method must be called after every call to env.reset.
```

Comment:
Could we modify this so the environment stores a variable containing the ticks per capture so that it doesn't have to be reset after each environment reset?